src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/index.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+
+import {
+  Radio,
+  Text,
+  Form,
+  Title,
+  FormGroup,
+  TextInput,
+  Gallery,
+  GalleryItem,
+  Button,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { AzureAuthButton } from './AzureAuthButton';
+import { AzureResourceGroups } from './AzureResourceGroups';
+import { AzureSourcesSelect } from './AzureSourcesSelect';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeAzureResourceGroup,
+  changeAzureShareMethod,
+  changeAzureSource,
+  changeAzureSubscriptionId,
+  changeAzureTenantId,
+  selectAzureResourceGroup,
+  selectAzureShareMethod,
+  selectAzureSubscriptionId,
+  selectAzureTenantId,
+} from '../../../../../store/wizardSlice';
+import { ValidatedTextInput } from '../../../ValidatedTextInput';
+import {
+  isAzureResourceGroupValid,
+  isAzureSubscriptionIdValid,
+  isAzureTenantGUIDValid,
+} from '../../../validators';
+
+export type AzureShareMethod = 'manual' | 'sources';
+
+// The Sources API only defines a V1ListSourceResponseItem[] type
+export type V1ListSourceResponseItem = {
```

Comment:
This is duplicate code, it's also present in `index.tsx` for the Aws step. Can we extract it from here and the AWS step into a new file? Perhaps `.../CreateImaveWizardV2/types.ts`?